inc/class-statify-dashboard.php

```diff
@@ -385,4 +385,24 @@ private static function _select_data() {
 
 		return $data;
 	}
+
+	/**
+	 * Parses a visit target and, if needed, modifies it for displaying.
+	 *
+	 * @param string $target URL target as displayed in widget.
+	 *
+	 * @return string
+	 */
+	public static function parse_target( $target ) {
+		// Modify for search strings.
+		if ( 0 === strpos( $target, '/?s=' ) ) {
```

Comment:
Are we sure the search parameter is always first, so there's no `...?foo=bar&s=searchstring`?
WP Core uses `isset( $this->query['s'] )` for that, so at this point it's position agnostic.

I'm fine with keeping it as simple and lightweight as possible here. But if that proves unreliable we might be better off extending this check or going the `wp_parse_url` / `wp_parse_args` way.

Reply:
oh, good point, will change that

Reply:
While working on that: we would also need to strip away any other query params except the search string, right (except the `p` query param, maybe)?

Reply:
Just tested it, no need for checking for the page query param, because if that exists, it overrules the search template, of course